src/components/FormFactory.vue

```diff
@@ -0,0 +1,144 @@
+<template lang="html">
+  <div id="form-factory">
+    <v-app>
+      <v-content>
+        <v-container fluid grid-list-md>
+          <v-layout row>
+            <v-flex xs6>
+              <v-form ref="myForm" v-model="formValid">
+                <v-jsonschema-form
+                  v-if="schema"
+                  :schema="schema"
+                  :model="dataObject"
+                  :options="options"
+                  @error="e => window.alert(e)"
+                  @change="change"
+                  @input="input"
+                >
+                  <template v-slot:prepend-fullKeySlot="{fullSchema}">
+                    Prepend slot<br>
+                  </template>
+                  <template v-slot:append-fullKeySlot="{fullSchema}">
+                    Append slot
+                  </template>
+                  <template v-slot:fullKeySlot="{fullSchema}">
+                    Full key slot: {{ fullSchema.description }}<br>
+                  </template>
+                  <template v-slot:custom-1="{fullSchema}">
+                    Custom display slot: {{ fullSchema.description }}
+                  </template>
+                </v-jsonschema-form>
+                <v-btn @click="handleClick">Add User</v-btn>
+                     <v-btn
+        color="error"
+        class="mr-4"
+        @click="reset"
+      >
+        Reset Form
+      </v-btn> 
+              </v-form>
+              <h2 class="title my-4" style="text-align: left;">
+                Data:
+              </h2>
+              <pre>{{ JSON.stringify(dataObject, null, 2) }}</pre>
+            </v-flex>
+          </v-layout>
+        </v-container>
+      </v-content>
+    </v-app>
+  </div>
+</template>
+
+<script>
+import VJsonschemaForm from '@koumoul/vuetify-jsonschema-form'
+//import examples from './examples'
+import hjson from 'hjson' // more tolerant parsing of the schema for easier UX
+import axios from 'axios'
+
+export default {
+  name: 'form-factory',
+  components: { VJsonschemaForm },
+  props: {
+    example: Object,
+  },
+  data: function() {
+    return {
+      window,
+      schema: null,
+      schemaStr: '{}',
+      schemaError: null,
+      dataObject: {},
+      //examples,
+      formValid: false,
+      options: null,
+      axios: axios,
+
+    }
+  },
+  mounted() {
+    if (window.location.search) {
+      //const key = window.location.search.replace('?example=', '')
+      //this.example = examples.find(e => e.key === key)
+    }
+    this.applyExample()
+  },
+  methods: {
+    applySchema() {
+      try {
+        this.schema = hjson.parse(this.schemaStr)
```

Comment:
Parsing with hjson will allow stuff to get created that will fail to parse on the server side. Can this parse be strict json instead?

Reply:
Yes, use JSON now and remove hjson dependency.